idl/github.com/uber/cadence/shared.thrift

```diff
@@ -842,6 +888,14 @@ struct SignalWorkflowExecutionRequest {
   30: optional string signalName
   40: optional binary input
   50: optional string identity
+  60: optional string requestId
+}
+
+struct DeleteWorkflowExecutionSignalRequest {
```

Comment:
Why do we need to define this on shared.idl?  This should not be public API.  It should only live on history.idl

Reply:
make sense, removed from shared.